docs/conf.py

```diff
@@ -146,6 +146,21 @@ def delete_attrs(cls):
         encoding="utf-8",
     )
     print(res)
+    # insert the Plausible analytics script to console.html
+    console_path = Path("_build/html/console.html")
+    console_html = console_path.read_text().splitlines(keepends=True)
+    for idx, line in enumerate(list(console_html)):
+        if 'pyodide.js">' in line:
+            # insert the analytics script after the `pyodide.js` script
+            console_html.insert(
+                idx,
+                '<script defer data-domain="pyodide.org" src="https://plausible.io/js/plausible.js"></script>\n',
```

Comment:
Maybe we can think of having two copies of `console.html` in the future, instead of doing complicated "replace" stuff. It will increase the maintenance burden, but we can remove all this stuff including PYODIDE_BASE_URL.

Reply:
yes, this is not ideal. Not sure about two copies as there is some amount of code there and it would be annoying to ask contributors to make the change twice. I think a better way is to use jinja2 templating more in the future, which would be easier once the main makefile is in python (or otherwise we have to use `jinja2-cli`).